less/about/error.less

```diff
@@ -12,7 +12,7 @@ body {
 
     .errorContent {
       display: flex;
-      width: 40vw;
+      width: 60vw;
```

Comment:
fyi this change affects every error page so let's not forget to do QA for the pages as well.